health/micro-ui/web/micro-ui-internals/packages/modules/campaign-manager/src/pages/employee/CreateChecklist.js

```
@@ -344,12 +378,19 @@ const CreateChecklist = () => {
 
   const payloadData = (data) => {
 
+    data.forEach((question) => {
+      if (question.type.code === "Short Answer") {
+        question.type.code = "String";
+        delete question.options;
+      }
+    });
+
     processedData = organizeQuestions(data);
     let { codes, local } = generateCodes(processedData);
```

Biome

[error] This let declares some variables that are only assigned once.

'codes' is never reassigned.

'local' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)